packages/gensx/src/models/environment.ts

```diff
@@ -0,0 +1,158 @@
+import { GenSXAPIResponse } from "../types/api.js";
+import { getAuth } from "../utils/config.js";
+import { USER_AGENT } from "../utils/user-agent.js";
+import { checkProjectExists } from "./projects.js";
+
+interface ListEnvironmentsResponse {
+  environments: {
+    id: string;
+    name: string;
+    projectId: string;
+    createdAt: string;
+    updatedAt: string;
+  }[];
+}
+
+/**
+ * List environments for a project
+ */
+export async function listEnvironments(
+  projectName: string,
+): Promise<ListEnvironmentsResponse["environments"]> {
+  const auth = await getAuth();
+  if (!auth) {
+    throw new Error("Not authenticated. Please run 'gensx login' first.");
```

Comment:
Can we just offer to log them in here instead of an error? Not sure which UX is better.

Reply:
I don't have a super strong opinion either way. I just followed the same convention that other commands used. 

Either way, if we make this update i'd prefer to save it for a separate PR.

Reply:
Sounds good